ethcore/private-tx/src/encryptor.rs

```diff
@@ -204,7 +202,7 @@ impl SecretStoreEncryptor {
 			Ok(accounts.sign(key_server_account.clone(), None, H256::from_slice(&contract_address_extended))?)
```

Comment:
Doesn't make sense to unlock then sign, just pass the password as second argument or if we need multiple passwords for whatever reason we should just try to sign in a loop.

Reply:
Reworked